energy-integration/common-modules/weekly-rewards-splitting/src/lib.rs

```diff
@@ -257,16 +257,47 @@ pub trait WeeklyRewardsSplittingModule:
         )
     }
 
+    #[view(getCurrentClaimProgress)]
+    fn get_current_claim_progress(
+        &self,
+        user: &ManagedAddress,
+    ) -> ClaimProgress<Self::Api> {
+        let claim_progress_mapper = self.current_claim_progress(user);
+        let is_new_user = claim_progress_mapper.is_empty();
+        if is_new_user {
+            ClaimProgress {
+                energy: Energy::default(),
+                week: 1,
```

Comment:
I think it should be
` ClaimProgress {
                energy: current_user_energy,
                week: current_week,
            }`